src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -1168,6 +1168,11 @@ private static string CleanFileName(string name, NamingConfig namingConfig)
 
         private string Truncate(string input, string formatter)
         {
+            if (input.IsNullOrWhiteSpace())
+            {
+                return string.Empty;
```

Comment:
I realized for release groups we want this to be null so we fall back to Sonarr, but not sure we should return null for other cases and an empty string may be better.

This will handle the release group scenario without forcing everything to null.

```csharp
tokenHandlers["{Release Group}"] = m => episodeFile.ReleaseGroup.IsNullOrWhiteSpace() ? m.DefaultValue("Sonarr") : Truncate(episodeFile.ReleaseGroup, m.CustomFormat);
```